terraform/custom-image/start

```diff
@@ -1,12 +1,15 @@
 #!/bin/sh
+ulimit -c unlimited
```

Comment:
can't we set it once in the terraform script?

Reply:
this works for current process current user only, otherwise we would need to re-login as user